link_shorterner/test/shortener_shortener_SUITE.erl

```diff
@@ -0,0 +1,71 @@
+-module(shortener_shortener_SUITE).
+
+-include_lib("common_test/include/ct.hrl").
+
+-export([all/0,
+         init_per_testcase/2,
+         end_per_testcase/2]).
+-export([not_existent_entries_get_notfound/1,
+         new_entries_get_created/1,
+         existent_entries_get_ok/1]).
+
+all() ->
+    [not_existent_entries_get_notfound,
+     new_entries_get_created,
+     existent_entries_get_ok
+    ].
```

Comment:
if you put the closing bracket in a newline (what i usually do), you should do the same with the starting bracket. on the other hand, be consistent. you can't start opening or closing brackets in a newline in a few places and not in the rest of the file/modules.